MEDMI_Data_Preprocessing/example_bash_script.sh

```diff
@@ -0,0 +1,26 @@
+#!/bin/bash
+
+# FILE_IN = 'inputs/temp_extras-rel_hum-pressure-dewpoint_June2017.csv'
+FILE_IN="inputs/temp_extras-rel_hum-pressure-dewpoint_midlands.csv"
+OUT_DIR='met_postprocessing'
+OUTFILE_SUFFIX='_midlands'
+STATIONS_FILENAME="../station_data/station_data_clean.csv"
+
+DATE_RANGE='2017-01-01_00 2017-12-31_23'
```

Comment:
Add extra speech marks around this, so that it is passed to python as a whole string:
DATE_RANGE='"2017-01-01 00 2017-12-31 23"'

Reply:
Ignore me - using the underscore is probably better.